test/studies/1brc/aggregators.perfkeys

```diff
@@ -0,0 +1 @@
+elapsed time: 
```

Comment:
I might add a [verify line](https://chapel-lang.org/docs/developer/bestPractices/TestSystem.html#validating-performance-test-output) to this too:
```
verify:-1:     İzmir: -18.6 17.9 54.0
```

Reply:
I think there's some amount of randomness to the input set (now that we're generating it dynamically) so verifying that way wouldn't work. Though we could rig it up to use some fixed seed if we wanted.